lib/ex_aws/utils.ex

```diff
@@ -100,4 +100,6 @@ defmodule ExAws.Utils do
 
     greg_secs - @seconds_0_to_1970
   end
+
+  def uuid, do: DateTime.utc_now |> :erlang.phash2
```

Comment:
Actually never mind, those can repeat over instances. I suppose concurrently creating hosted zones is unlikely so we'll stick with the current implementation.